projects/spectator/test/signal-input/signal-input.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { createComponentFactory, createHostFactory, Spectator, SpectatorHost } from '@ngneat/spectator';
+import { SignalInputComponent } from './signal-input.component';
+
+describe('SignalInputComponent', () => {
+  describe('with Spectator', () => {
+    let spectator: Spectator<SignalInputComponent>;
+
+    const createComponent = createComponentFactory({
+      component: SignalInputComponent,
+    });
+
+    beforeEach(() => {
+      spectator = createComponent({ props: { show: true } });
+    });
+
+    it('should render a SignalInputComponent', () => {
+      expect(spectator.query('#text')).toContainText('Hello');
+    });
+  });
+
+  describe('with SpectatorHost', () => {
```

Comment:
Let's add setInput calls here in these tests

Reply:
Good idea, aligned this with the equivalent `jest` test 🚀